Resources/Private/Language/de.FlexForms.xlf

```diff
@@ -1,71 +1,67 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 <xliff version="1.0">
-  <file source-language="en" datatype="plaintext" original="messages" date="2017-05-05T08:56:04Z"
-        target-language="de" product-name="service_bw2">
-    <header/>
-    <body>
+	<file source-language="en" datatype="plaintext" original="messages" date="2017-05-05T08:56:04Z"
+				target-language="de" product-name="service_bw2">
```

Comment:
indent

Reply:
Please all attributes in new line OR everything in one row.

Reply:
TYPO3 core put everything in one row, so I´ll go this way same to be consistent